src/main/java/uk/gov/justice/digital/delius/transformers/AssessmentTransformer.java

```diff
@@ -0,0 +1,49 @@
+package uk.gov.justice.digital.delius.transformers;
+
+import uk.gov.justice.digital.delius.data.api.OffenderAssessments;
+import uk.gov.justice.digital.delius.jpa.standard.entity.OASYSAssessment;
+import uk.gov.justice.digital.delius.jpa.standard.entity.OGRSAssessment;
+import uk.gov.justice.digital.delius.jpa.standard.entity.Offender;
+
+import java.time.LocalDate;
+
+public class AssessmentTransformer {
+
+    public static OffenderAssessments assessmentsOf(Offender offender, OGRSAssessment OGRSAssessment, OASYSAssessment OASYSAssessment) {
+        return OffenderAssessments
+            .builder()
+            .rsrScore(offender.getDynamicRsrScore())
+            .OGRSScore(getOGRSScore(OGRSAssessment, OASYSAssessment))
+            .build();
+    }
+
+    private static Integer getOGRSScore(OGRSAssessment OGRSAssessment, OASYSAssessment OASYSAssessment) {
+        Integer OGRSAssessmentScore = null;
```

Comment:
another nice thing if these end up being optionals we won't need the `if` statements since we can just `map` and `filter` and finish off with an `orElse`

Reply:
This bit is work in progress now, I'll have another go at it tomorrow morning.